M2/Macaulay2/d/actors3.d

```diff
@@ -2124,7 +2124,36 @@ toSequence(e:Expr):Expr := (
 	  else Expr(b.v)
 	  )
      is s:stringCell do Expr(strtoseq(s))
-     else WrongArg("a list, sequence, or string"));
+     else (
+	 iter := getIterator(e);
+	 if iter != nullE
+	 then (
+	     nextfunc := getNextFunction(iter);
+	     if nextfunc != nullE
+	     then (
+		 r := new Sequence len 1 do provide nullE;
+		 j := 0;
+		 y := nullE;
+		 while (
+		     y = applyEE(nextfunc, iter);
+		     when y
+		     is Error do return returnFromFunction(y)
+		     else nothing;
+		     y != StopIterationE)
+		 do (
+		     if j == length(r) then (
+			 r = new Sequence len 2 * length(r) do (
+			     foreach x in r do provide x;
+			     while true do provide nullE));
+		     r.j = y;
+		     j = j + 1);
+		 Expr(
+		     if j == 0 then emptySequence
+		     else if j == length(r) then r
+		     else new Sequence len j do (
+			 foreach x in r do provide x)))
+	     else buildErrorPacket("no method for applying next to iterator"))
+	 else WrongArg("a list, sequence, string, or iterable")));
```

Comment:
"iterable" is an adjective, so it doesn't match the other entries in this list.

Reply:
I borrowed the term "iterable" (as a noun) from Python here.  It's already in several other error messages,e.g.:

https://github.com/Macaulay2/M2/blob/fe53c0b3cd9ccb7ebac5786ecec466293a8f8502/M2/Macaulay2/d/evaluate.d#L327-L328

It's shorthand for "instance of a class with the `iterator` method installed".  It can a confusing term, though, so I'm totally fine with changing it.  Any suggestions?

Reply:
Let's fix all of those.  How about "iterable object"?

Reply:
Sounds good!  I'll push a new commit here in a bit.

Reply:
Done